azkaban-web-server/src/main/java/cloudflow/error/CloudFlowValidationException.java

```diff
@@ -1,25 +1,8 @@
 package cloudflow.error;
 
-import java.util.Arrays;
-import java.util.List;
-
 public class CloudFlowValidationException extends CloudFlowException {
 
-    // A list to avoid multiple api calls when there are various validation errors. Especially
-    // useful with endpoints that take several parameters.
-    private List<String> validationErrors;
```

Comment:
Supporting list of errors is too much work for the POC but we should definitely consider it in CloudFlow.